src/codeu/chat/server/Server.java

```diff
@@ -172,6 +163,44 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+    // Clean - A client wants to clean the log
+    this.commands.put(NetworkCode.CLEAN_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        File log = new File(persistentPath.getPath());
```

Comment:
on second thought, just pass the log path to controller and have the controller call the util function. I'm fine with both ways, since this is a very simple codebase, it's fine either way.